packages/payer-logo/PayerLogo.d.ts

```diff
@@ -1,8 +1,9 @@
 export interface PayerLogoProps {
     spaceId?: string;
     payerId?: string;
+    clientId?: string;
```

Comment:
If `clientId` is required then you don't have to add the `?` there.

Reply:
please see ec06bfa499f0f29937a48e5454b389dcc7be8eec

i have not delved into typescript yet 😄

Reply:
You're good, its new to me as well. Adding typings in is at minimum what we can do right now for `tsx` peeps.